syndesis/src/i18n/index.ts

```diff
@@ -0,0 +1,38 @@
+import i18n from 'i18next';
+import { en, it } from './locales';
+
+const options = {
+  interpolation: {
+    escapeValue: false, // not needed for react!!
+  },
+
+  debug: true, // change to false for release
+
+  resources: {
+    en: {
+      app: en.en.app,
+      dashboard: en.en.modules.dashboard,
+      shared: en.en.shared,
+    },
+    it: {
+      app: it.it.app,
+      dashboard: it.it.modules.dashboard,
+      shared: it.it.shared,
+    },
+  },
+
+  defaultNS: 'shared',
+  fallbackLng: 'it',
```

Comment:
Maybe protect this with a `process.env.NODE_ENV === 'production'` too, something like:

`fallbackLng: process.env.NODE_ENV === 'production' ? 'en' : 'it'`

Reply:
Done